First_project/First_project/Profile/ProfileHeaderView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ProfileHeaderView.swift
+//  First_project
+//
+//  Created by Taras Kyparenko on 21.03.2022.
+//
+
+import UIKit
+
+class ProfileHeaderView: UIView {
+    
+    private var profilePicture: UIImageView = {
+        var image = UIImage(named: "avatar")
+        var profilePicture = UIImageView(image: image)
+        profilePicture.layer.cornerRadius = 100 / 2
```

Comment:
You can make additional var in the class with the profilePicture width or height and then use it here and in the constraints